sample/src/main/scala/za/co/absa/spline/sample/batch/SampleJob2.scala

```diff
@@ -27,19 +27,19 @@ object SampleJob2 extends SparkApp("Sample Job 2") {
   val ds = spark.read
     .option("header", "true")
     .option("inferSchema", "true")
-    .csv("data/input/batch/wikidata.csv")
+    .csv("sample/data/input/batch/wikidata.csv")
```

Comment:
Why `sample/` prefix?

Reply:
To avoid having to change working directory in IDE.

Reply:
But it creates a tight dependency on the module directory name. Does it work if run from Maven?

Reply:
Good spot on the ability to run as described in the readme. As agreed I will separate these changes into new PR.